test/e2e/e2e.go

```diff
@@ -74,7 +75,7 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 			framework.Failf("Error deleting orphaned namespaces: %v", err)
 		}
 		framework.Logf("Waiting for deletion of the following namespaces: %v", deleted)
-		if err := framework.WaitForNamespacesDeleted(c, deleted, framework.NamespaceCleanupTimeout); err != nil {
+		if err := framework.WaitForNamespacesDeleted(c, deleted, 15*time.Minute); err != nil {
```

Comment:
Move wrong way, from configured value to hard-coded value, why is that?

Reply:
`NamespaceCleanupTimeout` is no longer getting exported.